birminghamio.ini

```diff
@@ -72,4 +72,6 @@ name=Trevor Morris
 [http://eclecticdreams.com/feed]
 name=Matt Machell
 [http://alexrussell.me.uk/atom.xml]
-name=Alex Russell
\ No newline at end of file
+name=Alex Russell
+[htpps://medium.com/feed/@PSDM]
```

Comment:
I suspect this should be https, not htpps? And... I think you've removed a bunch of the content of the file -- everything after your new insert seems to be missing, when compared with the current content of the file -- see the file in master, at https://github.com/BirminghamIO/planet.birmingham.io/blob/master/birminghamio.ini. Perhaps you're working from an older version or similar?